fix: add missing button attribute command & commandfor (#14761)
withastro/astro: .changeset/fifty-baboons-bow.md

---
'astro': patch
---

> Adds `command` and `commandfor` to the attribute that can be passed to the `button` element

Reviewer: ```suggestion
Updates `button` attributes types to allow `command` and `commandfor`
```
